### PR TITLE
Add a test workflow.

### PR DESCRIPTION
**Pull request by lucasmichot**
Add a Github test workflow.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 15
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```
Comment (atymic) [inline .github/workflows/test.yml:12]: May as well at 8 here

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          extension-csv: mbstring, xdebug
+
+      - name: Install dependencies
+        run: composer install --no-interaction --prefer-dist --no-suggest
+
+      - name: Lint composer.json
+        run: composer validate --strict
+
+      - name: Run Tests
+        run: vendor/bin/phpunit
```
Comment (atymic) [inline .github/workflows/test.yml:31]: look like we're missing codecov here as well?

**Review by atymic — COMMENTED** [review]
Should prob also remove the old travis job?

**Comment by lucasmichot** [comment]
> Should prob also remove the old travis job?

Done with 31cfca78b44b00cc70997fa31d2999ffc0cb5942

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
Looks like tests are failing, though they were before I think.

**Pull request merged**